app/src/main/java/io/gnosis/safe/ui/settings/app/passcode/ChangeCreatePasscodeFragment.kt

```diff
@@ -69,7 +75,7 @@ class ChangeCreatePasscodeFragment : BaseViewBindingFragment<FragmentPasscodeBin
         }
     }
 
-    private fun FragmentPasscodeBinding.skipPasscodeSetup() {
+    private fun skipPasscodeSetup() {
 
         findNavController().popBackStack(R.id.changePasscodeFragment, true)
         findNavController().currentBackStackEntry?.savedStateHandle?.set(SafeOverviewBaseFragment.OWNER_IMPORT_RESULT, false)
```

Comment:
why do we have this flag about owner import in change create fragment?

Reply:
It is only necessary to set this to false when coming from OwnerSelectionFragment. So it can be deleted here.

Reply:
There is more cleanup coming with the tests.